src/zkscript/fields/fq12_2_over_3_over_2.py

```diff
@@ -1,23 +1,41 @@
+"""Bitcoin scripts that perform arithmetic operations in a quadratic extension of F_q^6."""
+
 from tx_engine import Script
 
 from src.zkscript.util.utility_scripts import mod, nums_to_script, pick, roll, verify_bottom_constant
 
 
 class Fq12:
-    r"""F_q^12 as quadratic extension of F_q^6, which is built as cubic extension of F_q^2.
+    """Construct Bitcoin scripts that perform arithmetic operations in F_q^12 = F_q^6[u] / u^2 - v.
+
+    F_q^12 = F_q^6[u] / u^2 - v is a quadratic extension of F_q^6 = F_q^2[v] / v^3 - non_residue_over_fq2.
 
-    F_q^12 = F_q^6[u] / u^2 - v, F_q^6 = F_q^2[v] / v^3 - NON_RESIDUE_OVER_FQ2
+    Elements in F_q^12 are of the form `a + b * u`, where `a`, `b` are elements of F_q^6, `u^2` is equal
+    to `v`, and the arithmetic operations `+` and `*` are derived from the operations in F_q^6.
+
+    Attributes:
+        MODULUS: The characteristic of the field F_q.
+        FQ2: The script implementation of the field F_q^2.
```

Comment:
you mean how elements of fq2 are represented in script? I think I would like to know if this is a function, a class, ...